rpc/doc.md

```diff
@@ -182,9 +181,9 @@ Returns the information about cells collection by type_hash.
 
 ## Parameters
 
-    Type_hash - Cell type_hash.
-    From - Start BlockNumber.
-    To - End BlockNumber.
+    type_hash - Cell type_hash.
```

Comment:
```suggestion
    type_hash - Cell type hash.
```

The part after `-` is a description, I don't think we need the underscore in it. So does the following two lines.

Reply:
Nice, I'll update it.